subcommands/keys/rotate_root.go

```diff
@@ -34,12 +34,14 @@ func init() {
 	rotate.Flags().BoolVarP(&doRootSync, "sync-prod", "", false, "Make sure production root.json is up-to-date and exit")
 	rotate.Flags().BoolVarP(&initialRotation, "initial", "", false, "Used for the first customer rotation. The command will download the initial root key")
 	rotate.Flags().StringVarP(&changeReason, "changelog", "m", "", "Reason for doing rotation. Saved in root metadata for tracking change history")
+	rotate.Flags().StringP("key-type", "k", tufKeyTypeNameRSA, "Key type, supported: Ed25519, RSA (default).")
```

Comment:
i'd put this change and the one to rotate_targets.go into their own commit and i *think* we should probably default to ed25519.

good work.

Reply:
works for me.